src/TopPage/DataExtension.php

```diff
@@ -255,6 +255,11 @@ public function withFixedTopPage(int $topPageID, callable $callback)
         }
     }
 
+    public function updateCMSFields($fields)
```

Comment:
```suggestion
    public function updateCMSFields(FieldList $fields)
```
Needs to match the superclass's method signature
https://github.com/silverstripe/silverstripe-framework/blob/66561ccb49e196bd652b1fc2112c5aeda062354c/src/ORM/DataExtension.php#LL218C36-L218C55

Reply:
Updated